Header header
uint8[] image
float32 major_axis
float32 minor_axis
float32 orientation
